client/app/index.html

```diff
@@ -50,7 +50,14 @@
                         </div>
 
                         <!-- /Skip content target -->
-                        <ovh-chatbot></ovh-chatbot>
+                        <ovh-chatbot
+                            data-can-switch-to-livechat="true"
+                            data-config="{
+                                universe: 'WEB',
```

Comment:
You could use this no ? https://github.com/ovh-ux/ovh-manager-web/blob/master/client/app/app.js#L440